packages/foreman/foreman/foreman.spec

```
@@ -699,8 +700,8 @@ plugins required for Foreman to work.
   done
   # script content
   sed -ri 'sX/usr/bin/rakeX%{scl_rake}X' extras/dbmigrate script/foreman-rake
-
-  sed -i '/^ExecStart/ s|/usr/bin/sidekiq \(.\+\)$|/usr/bin/scl enable tfm "sidekiq \1"|' extras/systemd/%{dynflow_sidekiq_service_name}.service
+  # sidekiq wrapper
+  sed -i '/^exec/ s|/usr/bin/sidekiq "\(.\+\)"$|/usr/bin/scl enable tfm "/usr/bin/sidekiq \1"|' extras/%{name}-sidekiq
```

The whole point of the SCL wrapper is that you don't need to specify the full path. `/usr/bin/sidekiq` doesn't exist, but after activating the SCL env it is in `$PATH`.
```suggestion
  sed -i '/^exec/ s|/usr/bin/sidekiq "\(.\+\)"$|/usr/bin/scl enable tfm "sidekiq \1"|' extras/%{name}-sidekiq
```

However, this still removes the quotes from `"$@"` which I'm not sure actually works.

---

Honestly, I don't like solving artificial problems. We don't quote in the systemd unit either, there are no spaces and very unlikely will be. BUT since you asked:

```
[root@sat68 ~]# scl enable tfm -- ruby -e "puts :hello, :world"
hello
world

[root@sat68 ~]# scl enable tfm 'ruby -e "puts :hello, :world"'
hello
world

[root@sat68 ~]# scl enable tfm -- 'ruby -e "puts :hello, :world"'
/var/tmp/sclDm2GG6: line 8: ruby -e puts: command not found
```

I tend to prefer `--` syntax which I often use for git. I will refactor it this way.